openapi/src/parts/workspace.yaml

```diff
@@ -422,6 +431,9 @@ components:
           description: The ID of the workspace
           type: string
           format: uuid
+        userFacingId:
```

Comment:
nit: just to be tidy, I think this should be in the required list.

Reply:
Done